src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLike, color}) => {
+
+  const likedButtonContent = liked ? '❤️' : '🤍';
+  const localOrRemote = (sender === 'Vladimir') ? 'chat-entry local' : 'chat-entry remote';
```

Comment:
Nice. What would you have to add to your implementation if you didn't want to hard-code this sender?